cli/setup.py

```diff
@@ -44,7 +44,7 @@
         "httpx~=0.23.0",
         "msal >= 1.17.0",
         "jmespath==1.0.1",
-        "tabulate==0.8.10",
+        "tabulate==0.9.0",
```

Comment:
@tamirkamara tabulate was updated in the requirements.txt file in https://github.com/microsoft/AzureTRE/pull/3005 - but this file was not updated. Should it be updated here as well?

Reply:
and if so, I also noticed that requirements.txt has
`PyJWT==2.6.0` which is missing from here

Reply:
talked offline, the answer is yes